src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -62,3 +62,7 @@ settings.general.content.dimension.cap.max.height=Макс Высота (px)
 settings.general.content.dimension.cap.max.width=Макс Ширина (px)
 settings.general.smol.stickers.title=Маленькие Стикеры
 settings.general.smol.stickers.enable=Использовать Маленькие Стикеры
+notification.margin.saved.title=Sticker Position Saved
```

Comment:
@Dragon-0609 I did it again, could you please translate all of the newly added messages for me please? Thanks!

Reply:
Of course